specification/resourcemover/resource-manager/Microsoft.Migrate/stable/2023-08-01/resourcemovercollection.json

```diff
@@ -0,0 +1,2732 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Resource Mover Service API",
+    "description": "A first party Azure service orchestrating the move of Azure resources from one Azure region to another or between zones within a region.",
+    "version": "2023-08-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Migrate/moveCollections/{moveCollectionName}": {
+      "put": {
+        "tags": [
+          "MoveCollections"
+        ],
+        "operationId": "MoveCollections_Create",
+        "description": "Creates or updates a move collection.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/moveCollectionName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/MoveCollection"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
```

Comment:
Your descriptions "Success" result in the following linter error because they don't explain the difference between 200 and 201:

><html>
<body>
<!--StartFragment-->

❌ PutResponseSchemaDescription | Description of 200 response code of a PUT operation MUST include term 'update'.Location: Microsoft.Migrate/stable/2023-08-01/resourcemovercollection.json#L50
-- | --
❌ PutResponseSchemaDescription | Description of 201 response code of a PUT operation MUST include term 'create'.Location: Microsoft.Migrate/stable/2023-08-01/resourcemovercollection.json#L50

<!--EndFragment-->
</body>
</html>❌ [PutResponseSchemaDescription](https://github.com/Azure/azure-openapi-validator/blob/main/docs/put-response-schema-description.md)	Description of 200 response code of a PUT operation MUST include term 'update'.
Location: [Microsoft.Migrate/stable/2023-08-01/resourcemovercollection.json#L50](https://github.com/Azure/azure-rest-api-specs/blob/6f7c34a5f06fa06121119d4b952300c717b2d041/specification/resourcemover/resource-manager/Microsoft.Migrate/stable/2023-08-01/resourcemovercollection.json#L50)
❌ [PutResponseSchemaDescription](https://github.com/Azure/azure-openapi-validator/blob/main/docs/put-response-schema-description.md)	Description of 201 response code of a PUT operation MUST include term 'create'.
Location: [Microsoft.Migrate/stable/2023-08-01/resourcemovercollection.json#L50](https://github.com/Azure/azure-rest-api-specs/blob/6f7c34a5f06fa06121119d4b952300c717b2d041/specification/resourcemover/resource-manager/Microsoft.Migrate/stable/2023-08-01/resourcemovercollection.json#L50)